src/app/item-popup/ItemPopupBody.tsx

```diff
@@ -50,6 +52,27 @@ export default function ItemPopupBody({
     });
   }
 
+  const changeToTriageTab = useCallback((event: KeyboardEvent) => {
+    if (tabs.length < 2) {
+      return;
+    }
+    event.preventDefault();
+    onTabChanged(tabs[1].tab);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const changeToOverviewTab = useCallback((event: KeyboardEvent) => {
+    if (tabs.length < 2) {
+      return;
+    }
+    event.preventDefault();
+    onTabChanged(tabs[0].tab);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useHotkey('right', 'Switch to Triage tab', changeToTriageTab);
+  useHotkey('left', 'Switch to Overview tab', changeToOverviewTab);
```

Comment:
Love the functionality, but I was sorta reserving the arrow keys for a future feature where we can navigate selection of the item grid with the keyboard. I wonder what other keys would make sense here? Perhaps just a single key toggle?